cpp/src/io/orc/orc_gpu.hpp

```diff
@@ -287,6 +287,7 @@ void DecodeNullsAndStringDictionaries(ColumnDesc* chunks,
  * @param[in] num_rowgroups Number of row groups in row index data
  * @param[in] rowidx_stride Row index stride
  * @param[in] level Current nesting level being processed
+ * @param[out] error_count Number of errors during decode
```

Comment:
`DecodeOrcColumnData` is asynchronous. The fact that we copy `chunks` to host immediately after calling `DecodeOrcColumnData` should not impact how its implemented. If we return the error code we are enforcing this synchronization even though it might not be required otherwise.